datamade/ccfp-asset-dashboard
# Setup custom theming with Bootstrap

## Overview
This PR configures Bootstrap with Sass so that we can have more customize theming in the application. This PR also has some small UI changes to the portfolio planner.

### Demo
![ui-config-demo](https://user-images.githubusercontent.com/38969506/102794701-5caae080-4371-11eb-9ff8-8704e705b27f.gif)

### Notes
There is a tiny table refactor in here, which happened because I needed to add some custom selector components to the table rows.

## Testing Instructions
- Visit the review app: https://ccfp-asset-d-ui-styling-7jjaty.herokuapp.com/
- Make sure that you see the CCFP's dark green as a background color.
- Add/remove projects from the portfolio. The projects in the portfolio should be highlighted with a light green.
